compressor/tests/test_offline.py

```diff
@@ -58,7 +61,12 @@ def setUp(self):
         # template to be skipped over.
         django_template_dir = os.path.join(settings.TEST_DIR, 'test_templates', self.templates_dir)
         jinja2_template_dir = os.path.join(settings.TEST_DIR, 'test_templates_jinja2', self.templates_dir)
-        settings.TEMPLATE_DIRS = (django_template_dir, jinja2_template_dir)
+
+        if django.VERSION >= (1, 8):
+            self.override_template_dirs = self.settings(TEMPLATE_DIRS=(django_template_dir, jinja2_template_dir))
```

Comment:
Can't you use `self.settings` for < 1.8 too ? We have dropped support for django 1.3 so we should have `self.settings` and `override_settings` available for every version we support.

Even better, I wonder if you could clean up the `setUp` method to use `@override_settings` for every setting it changes, even the `TEMPLATE_DIRS`: instead of have a `templates_dir` property in each testcase, use  @override_settings(TEMPLATE_DIRS=func('thetemplatedir')) on each class ?